doc/Changelog.md

```diff
@@ -3,6 +3,8 @@ Changelog {#Changelog}
 
 # git master
 
+* [45](https://github.com/BlueBrain/Brion/pull/45):
+  Made targets optional
```

Comment:
Don't forget that you've also added the getNumNeurons() function in this commit.